nginx.conf

```diff
@@ -0,0 +1,117 @@
+user  nginx;
+worker_processes  auto;
+
+pid /var/run/nginx.pid;
+error_log stderr;
+events {
+}
+
+http {
+	# The mime type definitions included with nginx are very incomplete, so
+	# we use a list of mime types from the mailcap package, which is also
+	# used by most other Linux distributions by default.
+	include       /etc/nginx/mime.types;
+	# When recommendedOptimisation is disabled nginx fails to start because the mailmap mime.types database
+	# contains 1026 enries and the default is only 1024. Setting to a higher number to remove the need to
+	# overwrite it because nginx does not allow duplicated settings.
+	types_hash_max_size 4096;
+	
+	default_type application/octet-stream;
+	# optimisation
+	sendfile on;
+	tcp_nopush on;
+	tcp_nodelay on;
+	keepalive_timeout 65;
+	ssl_protocols TLSv1.2 TLSv1.3;
+	ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+	# Keep in sync with https://ssl-config.mozilla.org/#server=nginx&config=intermediate
+	ssl_session_timeout 1d;
+	ssl_session_cache shared:SSL:10m;
+	# Breaks forward secrecy: https://github.com/mozilla/server-side-tls/issues/135
+	ssl_session_tickets off;
+	# We don't enable insecure ciphers by default, so this allows
+	# clients to pick the most performant, per https://github.com/mozilla/server-side-tls/issues/260
+	ssl_prefer_server_ciphers off;
+	# OCSP stapling
+	ssl_stapling on;
+	ssl_stapling_verify on;
+	gzip on;
+	gzip_proxied any;
+	gzip_comp_level 5;
+	gzip_types
+	application/atom+xml
+	application/javascript
+	application/json
+	application/xml
+	application/xml+rss
+	image/svg+xml
+	text/css
+	text/javascript
+	text/plain
+	text/xml;
+	gzip_vary on;
+	# $connection_upgrade is used for websocket proxying
+	map $http_upgrade $connection_upgrade {
+		default upgrade;
+		''      close;
+	}
+	client_max_body_size 2G;
+	server_tokens off;
+	log_format logfmt 'time="$time_local" client=$remote_addr '
+	'method=$request_method url="$request_uri" '
+	'request_length=$request_length '
+	'status=$status bytes_sent=$bytes_sent '
+	'body_bytes_sent=$body_bytes_sent '
+	'referer=$http_referer '
+	'user_agent="$http_user_agent" '
+	'upstream_addr=$upstream_addr '
+	'upstream_status=$upstream_status '
+	'request_time=$request_time '
+	'upstream_response_time=$upstream_response_time '
+	'upstream_connect_time=$upstream_connect_time '
+	'upstream_header_time=$upstream_header_time';
+
+	server {
+		listen 0.0.0.0:80 default_server;
+		server_name thalia.nu;
+		location / {
+			proxy_pass http://concrexit:8000;
+		}
+		location /.well-known/change-password {
```

Comment:
There are more of these

Reply:
tru

Reply:
hmm the other ones aren't really useful for production simulation

Reply:
this one is also not that useful either :p